src/boolean3.cpp

```diff
@@ -179,14 +181,13 @@ struct Kernel11 {
 };
 
 struct Kernel02 {
-  VecView<const vec3> vertPosP;
-  VecView<const Halfedge> halfedgeQ;
-  VecView<const vec3> vertPosQ;
+  const Manifold::Impl& inA;
+  const Manifold::Impl& inB;
   const double expandP;
   VecView<const vec3> vertNormalP;
```

Comment:
we can get this from `inA`/`inB` and `forward`?

Reply:
Same as above.